compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/types/Types.java

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package io.ballerina.compiler.api.types;
```

Comment:
Why move this to `types` package as well? Wouldn't just having the builders in `types` do? Or rather, is it correct to group this in the same package as builders?

Reply:
@pubudu91, 
Packaged the Types.java abstract class in the `compiler.api` package. Moved the implementation of it (`BallerinaTypes`) into the `compiler.api.impl` package.